src/values/ObjectValue.js

```diff
@@ -1074,4 +1074,10 @@ export default class ObjectValue extends ConcreteValue {
   $OwnPropertyKeys(): Array<PropertyKeyValue> {
     return OrdinaryOwnPropertyKeys(this.$Realm, this);
   }
+
+  static refuseSerializationOnPropertyBinding(pb: PropertyBinding): boolean {
+    if (pb.object.refuseSerialization) return true;
+    if (pb.internalSlot && typeof pb.key === "string" && pb.key[0] === "_") return true;
```

Comment:
I'm not sure why all of these checks are necessary -- what are you trying to filter out vs leave in here?